### PR TITLE
Update Fedora defaults to 29

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: configs/mkosi.fedora [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: LGPL-2.1+
+# Let's build an image that is just good enough to build new mkosi images again
```
Comment (poettering) [inline configs/mkosi.fedora]: in systemd this dir is currently just called ".mkosi"...

I figure we should agree on a single name there, in particular as eventually I'd like to add "mkosi --all", that spiders through all such files and builds them all in one go.

Comment (poettering) [reply configs/mkosi.fedora]: i figure the dir name should have mkosi in the name. maybe "mkosi.recipes/"? or "mkosi.files/"?

Comment (keszybz) [reply configs/mkosi.fedora]: I like `mkosi.files` better. It's more generic.

Comment (keszybz) [reply configs/mkosi.fedora]: Updated.

**Pull request merged**